dwave/plugins/sklearn/transformers.py

```diff
@@ -0,0 +1,257 @@
+from sklearn.base import BaseEstimator
+from sklearn.feature_selection import SelectorMixin
+from dwave.system import LeapHybridCQMSampler
+import dimod
+import numpy as np
+import pandas as pd
+import tempfile
+
+import logging
+
```

Comment:
```suggestion

__all__ = ["HybridFeatureSelection"]

```

Subject to name comment below

Reply:
You might also consider importing it into the main namespace. 

Just add
```
from dwave.plugins.sklearn.feature_selection import *
```
in `dwave/plugins/sklearn/__init__.py`